src/test/java/com/rtb/projectmanagementtool/user/UserControllerTest.java

```diff
@@ -39,12 +39,18 @@ public void testAddUsers() {
         user2Majors.add("English");
         user2Majors.add("Religion");
         // Add 2 initially
-        ctl.addUser(
+        /* ctl.addUser(
             ds, new UserData(1l, 2l, "Sarah", 2022,
             user1Majors, Skills.LEADERSHIP.name(), 3));
         ctl.addUser(
             ds, new UserData(2l, 3l, "Joe", 2024,
-            user2Majors, Skills.ORGANIZATION.name(),5));
+            user2Majors, Skills.ORGANIZATION.name(),5)); */
+        ctl.addUser(
```

Comment:
This test still fails because "java.lang.Long cannot be cast to java.lang.Integer". I've tried casting each as it's respective type (long, int) in this assignment. I've also looked at UserController.java and UserData.java to ensure that I didn't use the incorrect order or mix up types there. Any tips/advice for how to proceed w/ this error?